lib/api/CDataFrameAnalysisInstrumentation.cc

```diff
@@ -64,10 +67,17 @@ const std::string PROGRESS_PERCENT{"progress_percent"};
 
 const std::size_t MAXIMUM_FRACTIONAL_PROGRESS{std::size_t{1}
                                               << ((sizeof(std::size_t) - 2) * 8)};
+
+std::int64_t bytesToMb(std::int64_t bytes) {
+    std::int64_t mb{static_cast<std::int64_t>(std::ceil(static_cast<double>(bytes) / 1048576))};
```

Comment:
I'm not 100% this wouldn't generate a warning for integer division by 1048576, but feels cleaner to just implement as integer division, i.e. this is equivalent to:
```suggestion
    return (bytes + 1048575) / 1048576);
```

Reply:
Oh, this is a nice trick!